orchagent/main.cpp

```diff
@@ -81,6 +81,8 @@ int32_t gVoqMaxCores = 0;
 uint32_t gCfgSystemPorts = 0;
 string gMyHostName = "";
 string gMyAsicName = "";
+uint64_t gSyncdRespTimeoutLC = ( 5 * SAI_REDIS_DEFAULT_SYNC_OPERATION_RESPONSE_TIMEOUT);
+uint64_t gSyncdRespTimeoutFab = ( 10 * SAI_REDIS_DEFAULT_SYNC_OPERATION_RESPONSE_TIMEOUT);
```

Comment:
Do we really need a cumulative timeout value here?  I find induvidually each asic/syncd switch_create takes less time in SUP and in LC. I will get and add few logs from device later.